### DISCUSSION TITLE
I just finish the first section of the course

### DISCUSSION THREAD

**Discussion by okekekingsley** [Cyfrin/foundry-full-course-cu#2665] [General]
Hi, my name is Kingsley, and I'm from Nigeria. I'm taking this course because I need a job in the near future and was fortunate enough to stumble upon it on YouTube. I consider myself a slow learner and sometimes lose focus while watching the videos, but I do my best to remember as much as I can. I'm really excited to dive into the world of smart contract development. Do you have any advice for a beginner with no coding experience on how to progress in this course and eventually land a job?

**Comment by Stargnite** [comment]
Hi Kingsley, my name is Tobi. I'm also from Nigeria. I just finished the beginner section of the course.
I'm also aspiring to be a smart contract developer. It's a pleasure to connect. Let go on this journey together🫱🏾‍🫲🏾

**Comment by wisdomtheprotector16** [comment]
Am Ibrahim Wisdom from Nigeria
I've been a block chain developer for 1 year now
Can we connect

**Reply by okekekingsley** [reply]
I would love to connect my Twitter handle is @9jaSmartConDev

**Reply by Stargnite** [reply]
Nice, just followed you @TobiSobayo1

**Reply by EngrPips** [reply]
Interesting

**Reply by Stargnite** [reply]
Can i get your X handle @wisdomtheprotector16 ?

**Reply by Heavens01** [reply]
Wow, lots of my country men from Nigeria here. I also just finished the first section of the course. I'm not that new to coding tho cause I'm a web developer. I aim to learn more skills for higher value and pay which has landed me here with you guys. I just pray we all just grab the concept of this course easily without difficulty...amen. I hope to connect with you all again when we get to another milestone. Love you all.

**Reply by Stargnite** [reply]
Amen... Thanks brother.

**Comment by EngrPips** [comment]
Interesting

**Comment by Kinglucky04** [comment]
hello Kingsley, my name is Lucky I am also from Nigeria, I have just concluded the basics too, but I am going through it over and over again so I will get a firm understanding of it.
I will advice you do same as I have realized that repetition helps you understand and recall better

**Reply by EngrPips** [reply]
You have spoken the truth.
